### ISSUE TITLE
Soundsync error on opening

### ISSUE THREAD

**Issue by danieljamesross** [geekuillaume/soundsync#36]
Mac OS 10.15.7

```Error: Not Found
    at Request.callback (/Applications/Soundsync.app/Contents/Resources/app/node_modules/superagent/lib/node/index.js:883:15)
    at IncomingMessage.<anonymous> (/Applications/Soundsync.app/Contents/Resources/app/node_modules/superagent/lib/node/index.js:1126:20)
    at IncomingMessage.emit (events.js:322:22)
    at IncomingMessage.EventEmitter.emit (domain.js:482:12)
    at endReadableNT (_stream_readable.js:1187:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)```

**Comment by geekuillaume** [comment]
I've seen these errors pops for some time and I finally found why. An error configuration on my server meant that for some requests, a 404 error message was returned. I fixed this but anyway this shouldn't crash Soundsync. I've made a change (https://github.com/geekuillaume/soundsync/commit/83fb569de81c5b1f84de436727c0113e514842ac) so that if the server is down, Soundsync will use other methods for connecting to every peers.
